src/rebuild/srv.c

```diff
@@ -1412,7 +1412,8 @@ rebuild_task_complete_schedule(struct rebuild_task *task, struct ds_pool *pool,
 					 task->dst_new_layout_version, &task->dst_tgts,
 					 retry_opc, 5);
 	} else if (task->dst_rebuild_op == RB_OP_REINT || task->dst_rebuild_op == RB_OP_EXTEND ||
-		   task->dst_rebuild_op == RB_OP_UPGRADE) {
+		   task->dst_rebuild_op == RB_OP_UPGRADE || task->dst_rebuild_op == RB_OP_EXCLUDE ||
+		   task->dst_rebuild_op == RB_OP_DRAIN) {
 		/* Otherwise schedule reclaim for reintegrate/extend/upgrade. */
```

Comment:
this comment need update as well.
But why need to schedule the RB_OP_RECLAIM task after rebuild done for RB_OP_EXCLUDE? and DRAIN?

Reply:
because some healthy shard might also be rebuilt after exclude/drain, due to co-located and rf issues, that is one of the reason we saw corruption in some cases.   See my test in this PR.